packages/manager/modules/pci/src/projects/project/private-registry/onboarding/index.js

```diff
@@ -0,0 +1,66 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import component from './onboarding.component';
+
+const moduleName = 'ovhManagerPciPrivateRegistryOnboardingLazyLoading';
```

Comment:
We should have one module `ovhManagerPciPrivateRegistryOnboardingLazyLoad` to `lazyload` the  `ovhManagerPciPrivateRegistryOnboarding` module